.kitchen.yml

```diff
@@ -31,6 +31,15 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
+# Solaris 10 requires the OS ISO to be mounted into `/cdrom` in order for the
```

Comment:
Yeah you can ship the ISO and set it up to mount. It might be easier to just copy the required packages to `/var/spool/pkg` on the box, as that is the default source location for package installations on Solaris 10.